src/routers/course.ts

```diff
@@ -1,24 +1,65 @@
 import express from 'express';
+import { create, read, update, del } from '../operators/courseOperations';
 
 const courseRouter = express.Router();
+const app = express();
+
+app.use(express.json());
```

Comment:
I think these need to be removed since this is potentially overwriting the `app` defined in `src/index.ts`

Reply:
Correct, we only need to do this once in the src/index.ts